vignettes/VISION-vignette.Rmd

```diff
@@ -172,3 +172,78 @@ projection <- read.csv("umap_results.csv")
 
 vis <- addProjection(vis, "UMAP", projection)
 ```
+
```

Comment:
I think the name of the expression matrix file for this vignette is wrong (should be matrix instead of counts?)

Reply:
Good catch!